src/services/venueService.ts

```diff
@@ -0,0 +1,18 @@
+import { type Venue } from '@prisma/client'
+import prisma from '../config/prisma/prisma-client'
+
+
+export async function getAllVenues(): Promise<Venue[]> {
+  return prisma.venue.findMany()
+}
+
+
+export async function getVenueById(venueId: string) {
+    return prisma.venue.findUnique({
```

Comment:
Just add TypeScript here on return of function.. promise<venue>

Reply:
//TODO find return type for this function 

return is breaking with : Promise<Venue>